coderd/rbac/astvalue.go

```diff
@@ -81,6 +81,10 @@ func (s Subject) regoValue() (ast.Value, error) {
 		return nil, xerrors.Errorf("expand scope: %w", err)
 	}
 	subj := ast.NewObject(
+		[2]*ast.Term{
+			ast.StringTerm("type"),
+			ast.StringTerm(string(s.Type)),
+		},
```

Comment:
I don't think we need to include this. This is for passing objects to the rego OPA authz policy files, and since the type isn't consumed by those policy files it's not necessary